I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/custom_screens.rpy", line 18, in script
    call screen MapUI
ScriptError: could not find label 'house2_pressed'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\Ian\Downloads\renpy-7.4.11-sdk/renpy/bootstrap.py", line 331, in bootstrap
    renpy.main.main()
  File "C:\Users\Ian\Downloads\renpy-7.4.11-sdk/renpy/main.py", line 662, in main
    run(restart)
  File "C:\Users\Ian\Downloads\renpy-7.4.11-sdk/renpy/main.py", line 148, in run
    renpy.execution.run_context(True)
  File "C:\Users\Ian\Downloads\renpy-7.4.11-sdk/renpy/execution.py", line 922, in run_context
    context.run()
  File "game/custom_screens.rpy", line 18, in script
    call screen MapUI
  File "C:\Users\Ian\Downloads\renpy-7.4.11-sdk/renpy/script.py", line 918, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'house2_pressed'.

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
CS 1.0
Mon Feb 14 02:31:48 2022
